.github/workflows/release.yml

```diff
@@ -150,14 +151,29 @@ jobs:
           git commit -am "docs: release ${{ needs.process-inputs.outputs.MAJOR_MINOR }}"
           # Clean up auxiliary files, so next steps run on a clean tree
           git clean -fx :/
-      - name: Download updated contrast releases file (from release branch)
+      - name: Download artifacts
         uses: actions/download-artifact@65a9edc5881444af0b9093a5e628f2fe47ea3b2e #v4.1.7
         with:
-          name: contrast-releases.json
-          path: ./contrast-main/packages/contrast-releases.json
+          merge-multiple: true
```

Comment:
Will this always download all artifacts? We should document the problem we faced here and link to the issue.